phoenix-core/src/main/java/org/apache/phoenix/schema/PTableImpl.java

```diff
@@ -1445,6 +1446,35 @@ public Integer getBucketNum() {
         return indexes;
     }
 
+    @Override
+    public List<PTable> getIndexes(boolean viewOnly) {
```

Comment:
I think a comment block explaining the need for 'viewOnly' here would be useful. What does the function return when it's true vs false.

Reply:
Yes, I agree with you. For the consistency with rest of the code, as this is an overridden method, the comments for all overridden methods are provided in the base class, in this case, PTable.java.